### PR TITLE
How to run multiple instances with docker

### PR DESCRIPTION
**Pull request by Tako88**
Basic guide on how to run multiple instances using docker.

## Summary

To add information about running multiple instances of freqtrade with docker.

### CODE REVIEW

File: docs/advanced-setup.md [hunk 1]
```diff
@@ -52,6 +52,71 @@ freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite:///user
...
+    #   dockerfile: "./docker/Dockerfile.custom"
+    restart: always
+    container_name: freqtrade2
+    volumes:
+      - "./user_data:/freqtrade/user_data"
+    # Expose api on port 8080 (localhost only)
+    # Please read the https://www.freqtrade.io/en/latest/rest-api/ documentation
+    # before enabling this.
+    ports:
+      - "127.0.0.1:8081:8081"
```
Comment (xmatthias) [inline docs/advanced-setup.md]: I'd think that a better way (especially when using modular configs) is to use the following port mapping.
This way, the bot still litens on 0.0.0.0:8080, but you connect via 127.0.0.1:8081.


```suggestion
      - "127.0.0.1:8081:8080"
```

To me, this is easier, but it requires pretty good understanding of this concept - so i'd leave this up to you...

File: docs/advanced-setup.md [hunk 2]
```diff
@@ -52,6 +52,71 @@ freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite:///user
 
 For more information regarding usage of the sqlite databases, for example to manually enter or remove trades, please refer to the [SQL Cheatsheet](sql_cheatsheet.md).
 
+### Multiple instances using docker
+
+To run multiple instances of freqtrade using docker you will need to edit the docker-compose.yml file and add all the instances you want as separate services. Remember, you can separate your configuration into multiple files, so it's a good idea to think about making them modular, then if you need to edit something common to all bots, you can do that in a single config file. 
+```
```
Comment (xmatthias) [inline docs/advanced-setup.md]:
```suggestion
``` yml
```

File: docs/advanced-setup.md [hunk 3]
```diff
@@ -52,6 +52,71 @@ freqtrade trade -c MyConfigUSDT.json -s MyCustomStrategy --db-url sqlite:///user
...
+      trade
+      --logfile /freqtrade/user_data/logs/freqtrade2.log
+      --db-url sqlite:////freqtrade/user_data/tradesv3_freqtrade2.sqlite
+      --config /freqtrade/user_data/config.json
+      --config /freqtrade/user_data/config.freqtrade2.json
+      --strategy SampleStrategy
+
+```
+You can use whatever naming convention you want, freqtrade1 and 2 are arbitrary.
+
```
Comment (xmatthias) [inline docs/advanced-setup.md:118]: Please add a note here which states that the above restrictions (different databases, different telegram bots) still applies

File: docs/advanced-setup.md (hunk 1 above)
Comment (Tako88) [reply docs/advanced-setup.md]: Oh good catch, agreed.

**Review by xmatthias — APPROVED** [review]
LGTM :+1:

**Pull request merged**